### PR TITLE
Guest user visits root

### PR DESCRIPTION
**Pull request by karinamzalez**
* feature test written: when guest user goes to root, they see all businesses
* incorporated slug usage in test to encourage adding that sooner rather than later
* we should consider upgrading to rails 5 to lose the deprecation warnings. 
* added bootstrap-- subject to change.

### CODE REVIEW

File: .rspec [hunk 1]
```diff
@@ -1,2 +1,3 @@
 --color
 --require spec_helper
+--format NyanCatFormatter
```
Comment (zackforbing) [inline .rspec]: NYAN CAT YESSSS

**Comment by zackforbing** [comment]
looks good, ready to merge

File: spec/features/guest_user_visits_home_page_spec.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.feature "Guest User Sees All Businesses" do
```
Comment (zackforbing) [inline spec/features/guest_user_visits_home_page_spec.rb]: let's use "visitor" instead of "guest user"

**Comment by kbs5280** [comment]
Pull request looks ready to merge.

**Pull request merged**